src/screens/DiscoveryScreenV2/elements/DiscoveryTitleSeparator.js

```diff
@@ -20,7 +20,7 @@ const DiscoveryTitleSeparator = (prop) => {
         fontSize: 12,
         lineHeight: 18,
         fontFamily: fonts.poppins[600],
-        color: COLORS.white2,
+        color: COLORS.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color has been changed from `COLORS.white2` to `COLORS.white`. Ensure that this change is intentional and does not affect the readability or overall design of the UI. If `COLORS.white2` was a softer variant of white, changing it to pure white might make the text too bright on some screens.
```diff
-        color: COLORS.white2,
+        color: COLORS.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->